cardstack/cardstack
# Create Link and CTA core types

closes #1240 
closes #1241 

After merge, we need to rev the hub.

## Added
- Link core type
- CTA core type
- tests for CTA and Link
- Styling for the Link core type
- A new color to the palette (in ui-components and variables)

## Changed
- Switched some former urls to be actual URL types

## How to test
Create a card, drop in a CTA and Link field using the schema editor, fill in the URLs in edit mode, and see how it looks in layout
